repspark_at/actions/setup_class.py

```diff
@@ -0,0 +1,22 @@
+from appium import webdriver
+
+class setUpClass('???????????????'):
+    #@classmethod  I'm not sure if I should use classmethod here
```

Comment:
[Here](https://docs.python.org/3/library/unittest.html#unittest.TestCase.setUpClass) is the answer.